### PR TITLE
Describe how to cancel order with items from backorders salable qty

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request describes how to cancel order with items from backorders salable qty

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

https://docs.magento.com/user-guide/catalog/inventory-backorders.html?itm_source=merchdocs&itm_medium=quick_search&itm_campaign=federated_search&itm_term=back

whatsnew
Updated the [Inventory Management - Configuring Backorders](https://docs.magento.com/user-guide/catalog/inventory-backorders.html) topic to include a note to clarify how canceled orders affect the salable quantity.

### CODE REVIEW

File: src/catalog/inventory-backorders.md [hunk 1]
```diff
@@ -37,6 +37,9 @@ _Example Salable Quantity after backorders enabled_
 
 When customers purchase more than 25 products, new orders enter as backorders. As the product's Salable Quantity reduces to 5 (70 items have been sold), the _Product_ page will display a message `Only 5 left` on the storefront. When the Salable Quantity reaches `0`, the product is displayed as `Out of Stock` in the storefront.
 
+{:.bs-callout-info}
+When a customer places an order using backorder qty, Inventory Management automatically subtracts qty from the salable quantity. If an order has not been shipped and is canceled the quantity returns to the aggregated virtual salable quantity. Important to note, that the canceled order quantity is not assigned to any of the sources, but is returned to the total number of products available for sale, in column `Salable Quantity` on the product grid.
```
Comment (leslietilling) [inline src/catalog/inventory-backorders.md:41]:
```suggestion
When a customer places an order using _backorder qty_, Inventory Management automatically subtracts the quantity from the salable quantity. If an order is not shipped and is canceled, the quantity returns to the aggregated virtual salable quantity. The **_canceled order quantity is not assigned to any of the sources_**, but is returned to the total number of products available for sale (_Salable Quantity_ column  on the products grid).
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Proposed edits for style and clarity

**Comment by ZhovnirMaria** [comment]
@leslietilling please check, thanks ☺️

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**